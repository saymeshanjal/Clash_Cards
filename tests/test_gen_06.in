10
4 20 2
16 7 4
2 28 3
16 11 5
16 19 5
16 5 5
14 14 3
16 47 3
14 23 2
8 48 1
