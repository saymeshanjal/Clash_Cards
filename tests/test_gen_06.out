1 2 3
9 11331 1530
0 2 0
78 12792 0
235 12635 0
0 12870 0
448 2984 0
3214 9656 0
83 394 2955
35 35 0
